docs/mast/mast.rst

```diff
@@ -503,14 +503,71 @@ Individual or ranges of spectra can be downloaded using the `~astroquery.mast.Ca
                 ./mastDownload/HSC/HAG_J072704.73+691808.0_J8HPAOZMQ_V01.SPEC1D.fits ... None
                 
 
-
 Accessing Proprietary Data
 ==========================
 
 To access data that is not publicly available users may log into their
 `MyST Account <https://archive.stsci.edu/registration/index.html>`_.
 This can be done by using the `~astroquery.mast.MastClass.login` function,
-or by initializing a class instance with a username/password.
+or by initializing a class instance with credentials.
+
+MAST is in the process of upgrading our Auth infrastructure to a token based system.
+This will be deployed early November 2018, at which point the credentials will
+switch from accepting a username/password and instead accept a auth token.
```

Comment:
a -> an

may be also change the future tense here given by the time of merging it's almost happened. Not a blocker again.